edweiss-app/components/core/modal/ModalContainer.tsx

```diff
@@ -42,6 +42,37 @@ const ModalContainer: ReactComponent<{ snapPoints?: (string | number)[], modalRe
 
 export default ModalContainer;
 
+export const ModalContainerScrollView: ReactComponent<{ snapPoints?: (string | number)[], disabledDynamicSizing?: boolean, modalRef: React.RefObject<BottomSheetModalMethods>, children?: ReactNode; }> = (props) => {
+	const backgroundColor = useColor("mantle");
+	const handleIndicatorColor = useColor("base");
+
+	const { handleSheetPositionChange } = useBottomSheetBackHandler(props.modalRef);
+
+	const renderBackdrop = useBackdrop();
+
+	const snapPoints = props.snapPoints ?? [];
+
+	return (
+		<BottomSheetModal
+			ref={props.modalRef}
+			index={0}
+			snapPoints={snapPoints}
+			backdropComponent={renderBackdrop}
+
+			enableDynamicSizing={!props.disabledDynamicSizing}
+
+			enableContentPanningGesture={false}
+			handleIndicatorStyle={{ backgroundColor: handleIndicatorColor }}
+			backgroundStyle={{ backgroundColor }}
+			onChange={handleSheetPositionChange}
+		>
+			<BottomSheetScrollView>
+				{props.children}
+			</BottomSheetScrollView>
+		</BottomSheetModal>
+	);
+};
+
 // export function ModalContainerScrollView(props: { snapPoints?: (string | number)[], modalRef: React.RefObject<BottomSheetModalMethods>, children?: ReactNode }) {
 // 	const backgroundColor = useColor("mantle");
 // 	const handleIndicatorColor = useColor("base");
```

Comment:
I think, you can delete this useless comments

Reply:
Oops, I will.